src/commands/info.js

```diff
@@ -11,8 +11,8 @@ class InfoCommand extends ApifyCommand {
             const niceInfo = {
                 username: info.username,
                 userId: info.id,
-                token: info.token,
-                proxyPassword: info.proxy.password,
+                token: `******${info.token.slice(-4)}`,
+                proxyPassword: `******${info.proxy.password.slice(-4)}`,
```

Comment:
Why showing even a bit of password? I just wouldn't show it at all.

Reply:
It was there just for user that he should see that we saved these tokens for him and he can check if token are right by 4 last digit.